website/www/site/content/en/powered-by/open-source/scio.md

```diff
@@ -0,0 +1,20 @@
+---
+title:  "Scio"
+type: open-source
+categories:
+  - Open Source Projects Powered by Apache Beam
+icon: /images/logos/powered-by/scio.png
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
```

Comment:
Do we have any copy for Scio?

Reply:
unfortunately no

Reply:
@nevillelyh is there any chance you or someone else involved with Scio provide a blurb about it for our new "powered by" card? For reference the blurb for Klio is:
> Developed at Spotify and built on top of Apache Beam for Python, Klio is an open source framework that lets researchers and engineers build smarter data pipelines for processing audio and other media files, easily and at scale.

Reply:
Yesterday, I got something like this: 
`Developed at Spotify and built on top of Apache Beam, Klio is an open source framework for data processing pipelines for audio and other media files.`

Reply:
That is for Klio, a different, similarly named library from Spotify :)

Reply:
> Another issue I just noticed in the [staged website](http://apache-beam-website-pull-requests.storage.googleapis.com/13858/index.html), the footer is centered:
> ![image](https://user-images.githubusercontent.com/675055/106629767-057f5a80-6530-11eb-8851-85af2aa86d7e.png)

Merge of website-revamp fixed this

Reply:
Let's file a jira to make sure we get some copy for Scio at some point